Source/WebKit/UIProcess/Extensions/Cocoa/WebExtensionContextCocoa.mm

```diff
@@ -309,7 +310,10 @@ static WKWebExtensionContextError toAPI(WebExtensionContext::Error error)
 
 NSArray *WebExtensionContext::errors()
 {
-    return [protectedExtension()->errors() arrayByAddingObjectsFromArray:m_errors.get()];
+    auto array = createNSArray(protectedExtension()->errors(), [](Ref<API::Error> error) {
+        return ::WebKit::wrapper(error);
+    });
```

Comment:
```suggestion
    auto *array = createNSArray(protectedExtension()->errors(), [](Ref<API::Error> error) {
        return wrapper(error);
    });
```

Reply:
`error: variable 'array' with type 'auto *' has incompatible initializer of type 'RetainPtr<NSMutableArray>'`

Reply:
Ah, always forget it returns `RetainPtr`.